### PR TITLE
Add in Yard doc

### PR DESCRIPTION
**Pull request by joesustaric**
Yard doc it up!

### CODE REVIEW

File: lib/event_sourcery/postgres/config.rb [hunk 1]
```diff
@@ -26,14 +26,23 @@ def initialize
...
+      # The event source.
+      #
+      # @return [EventSource]
       def event_source
         @event_source ||= ::EventSourcery::EventStore::EventSource.new(event_store)
       end
 
+      # The event sink.
+      #
+      # @return [EventSink]
```
Comment (orien) [inline lib/event_sourcery/postgres/config.rb]: Comments like these add no value. No value is provided over just reading the method name. I think we should remove them.

File: lib/event_sourcery/postgres/reactor.rb [hunk 2]
```diff
@@ -11,18 +11,31 @@ def self.included(base)
...
+        # Assign the types of events this reactor can emit.
+        #
+        # @param event_types the types of events this reactor can emit
         def emits_events(*event_types)
           @emits_event_types = event_types
         end
 
+        # Returns an array of which event types this reactor can emit.
+        #
+        # @return [Array] an array of the types of events this reactor can emit
```
Comment (orien) [inline lib/event_sourcery/postgres/reactor.rb]: This comment is duplicated. Can we just go with the `@return` line?

File: lib/event_sourcery/postgres/tracker.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 module EventSourcery
   module Postgres
     class Tracker
+      # This will set up a persisted event id tracker for processors.
```
Comment (orien) [inline lib/event_sourcery/postgres/tracker.rb]: Is this describing the `Tracker` class? If so it should be above the `class Tracker` line.

File: lib/event_sourcery/postgres/version.rb [hunk 4]
```diff
@@ -1,5 +1,6 @@
 module EventSourcery
   module Postgres
+    # Defines the version
```
Comment (orien) [inline lib/event_sourcery/postgres/version.rb]: This is not helpful.

**Comment by joesustaric** [comment]
Thanks for the feedback @orien . Made the changes you suggested. I'm keen just to merge a first version of the doco in. We can elaborate and improve upon it as we go :) Thanks

**Comment by orien** [comment]
Cheers @joesustaric. I'm a little concerned we're adding comments for the sake of adding comments. Unhelpful comments are worse then no comments, they slow developers down. I'd prefer to only let the helpful stuff in.

**Comment by joesustaric** [comment]
@orien yup agreed. Are you happy to merge this first version in for now? Also, should I add a note in the unreleased section? (I think maybe not, there is no new/deleted functionality with this PR)

**Comment by joesustaric** [comment]
@orien / @stevehodgkiss any objections in merging this first version in?

**Pull request merged**